conf/utils.py

```diff
@@ -39,7 +39,7 @@ def getpythonpid(process_name):
 
 def get_json_conf(path, dump):
     try:
-        with open(path, 'r') as f:
+        with open(path, "r") as f:
```

Comment:
Sorry about this - automatic formatter. Shall I revert?

Reply:
No, please leave it. I applied formatting to the whole Python code a while ago (#748) but it looks like I unintentionally introduced the single quotes in a later PR